package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@safe-global/safe-contracts",
-    "version": "1.4.1-build.0",
+    "version": "1.4.1-1",
```

Comment:
Nit: can you bump the version in the `README` as well? I think it still points to `-build.0`.

Reply:
@nlordell, it references a commit hash of the release, but I cannot really know it in advance, maybe there's a workaround though 😄 that's why I didn't change it

Reply:
**Edit**: As @mmv08 - referencing a commit hash is self-referencial and not really possible. The workaround is to reference a tag by name instead.